src/components/MatPassContent.tsx

```
@@ -47,124 +67,180 @@ interface CartItem {
   price: number;
   quantity: number;
   isPass?: boolean;
+  isAcro?: boolean;
   productType?: string;
   bundleTypeId?: string;
   uniqueId: string;
 }
 
 const getPackageNumber = (name: string): number => {
-  const number = parseInt(name.split(' ')[0] || '1', 10);
+  const number = parseInt(name.split(" ")[0] || "1", 10);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)